### ISSUE TITLE
please support Facebook SDK v17.0.0

### ISSUE THREAD

**Issue by command-not-found** [darwin-morocho/flutter-facebook-auth#395]
### What version are you using?

flutter_facebook_auth: ^6.1.1

### What OS and version are you using to local deploy your application?

Mac 14.4.1

### What platforms are you seeing the problem on?

iOS

### pubspec.yaml

```yaml
flutter_facebook_auth: ^6.1.1
```

### Describe the Bug

The latest version 17.0.0 is required to support Apple's privacy policy

### Expected Behavior

Update to 17.0.0 

### To Reproduce

<img width="1242" alt="17" src="https://github.com/darwin-morocho/flutter-facebook-auth/assets/17892680/c86a2e08-a52c-487f-bbda-3aea5a45ce82">

### flutter doctor -v

```shell
Flutter (Channel stable, 3.19.2, on macOS 14.4.1 23E224 darwin-arm64, locale zh-Hans-CN)
    • Flutter version 3.19.2 on channel stable at /Users/chunlinwang/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7482962148 (5 weeks ago), 2024-02-27 16:51:22 -0500
    • Engine revision 04817c99c9
    • Dart version 3.3.0
    • DevTools version 2.31.1
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn
```

**Comment by aggeloskoutanis** [comment]
+1

**Comment by antonengelhardt** [comment]
+1 !!

**Comment by antonengelhardt** [comment]
```
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 17.0)

    flutter_facebook_auth (from `.symlinks/plugins/flutter_facebook_auth/ios`) was resolved to 6.0.4, which depends on
      FBSDKLoginKit (~> 16.3.1) was resolved to 16.3.1, which depends on
        FBSDKCoreKit (= 16.3.1)
```

**Comment by RoiLaredo** [comment]
+1

**Comment by parajuliamit** [comment]
+1

**Comment by mkobuolys** [comment]
+1. This one must be resolved. Otherwise, app updates won't be approved after May 1.

**Comment by darwin-morocho** [comment]
Fixed on `flutter_facebook_auth: 6.2.0`
